### ISSUE TITLE
Syntax highlighting inside jldoctest blocks

### ISSUE THREAD

**Issue by waldyrious** [JuliaEditorSupport/julia-vscode#251]
Not sure this is possible or even desirable for others, but it would be nice to be able to visually parse the Julia code in documentation blocks more easily. Perhaps somewhat faded out so it doesn't mix up with real code?

**Comment by davidanthoff** [comment]
Yeah, that would be fantastic. Not sure how one would achieve that, though, we would need someone who understands the syntax files really well to do that.
